### PR TITLE
Corrected link

### PR DESCRIPTION
**Pull request by alex-lush**
Did point to Exchange Online PowerShell connection details, corrected to point to Security and Compliance Centre PowerShell.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/retention-policies.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ Therefore, before you lock a retention policy, it's **critical** that you unders
   
 You can lock a retention policy only by using PowerShell.
 
-First, [connect to Office 365 Security & Compliance Center PowerShell](https://go.microsoft.com/fwlink/p/?LinkID=799771).
+First, [connect to Office 365 Security & Compliance Center PowerShell](https://docs.microsoft.com/en-us/powershell/exchange/office-365-scc/connect-to-scc-powershell/connect-to-scc-powershell?view=exchange-ps).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-policies.md]:
```suggestion
First, [connect to Office 365 Security & Compliance Center PowerShell](https://docs.microsoft.com/powershell/exchange/office-365-scc/connect-to-scc-powershell/connect-to-scc-powershell?view=exchange-ps).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - please remove locale from link.

**Comment by mypil** [comment]
@cabailey - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Comment by cabailey** [comment]
#sign-off Confirmed link

**Pull request merged**